### PR TITLE
New "Bicycle Dynamics for Microscopic Traffic Simulation" project page.

### PR DESCRIPTION
**Pull request by chrismo-schmidt**
I finally wrote the project page for my PhD project. The featured images are already uploaded and the page builds correctly locally. 

I also adapted Anna's and my topic names on the member page to align better with what we are doing. 

Finally, I also added a few things to .gitignore to keep the repo clean from any files of my local build. Pls remove if this is unwanted.

### CODE REVIEW

File: content/pages/research/bicycle-dynamics-microsim.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+- **Native Cyclist Social Forces for SUMO:** Adding a new C++ implementation of our cyclist 
+  social force model to the codebase of DLR's open-source simulator SUMO_ to make the model 
+  generally available for use in traffic simulation.   
+  
+
+
+Research Output
+===============
+
+:Journal Articles:
```
Comment (moorepants) [inline content/pages/research/bicycle-dynamics-microsim.rst:86]: I"ve never seen this rst notation. What does it do?

**Comment by moorepants** [comment]
One issue, you've written directly to your source branch so I can't clone it without writing over my source branch. You should always create a new branch (branched from the latest source branch) then add your changes.

File: content/pages/research/bicycle-dynamics-microsim.rst (hunk 1 above)
Comment (moorepants) [reply content/pages/research/bicycle-dynamics-microsim.rst:86]: Some formatting issue:
image

**Comment by moorepants** [comment]
This looks great! If you fix that formatting issue we can merge.

File: content/pages/research/bicycle-dynamics-microsim.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+	Schmidt, C. M., Dabiri, A., Schulte, F., Moore, J. K. & Happee, R. (2024). Cycling Safety Assessment in Microscopic Traffic Simulation: A Review and Methodological Framework [Manuscript submitted to Transportation Research Interdisciplinary Perspectives].
+
+:Conference Papers:
+
+	Schmidt, C. M., Dabiri, A., Schulte, F., Happee, R., & Moore, J. K. (2024). Essential Bicycle Dynamics for Microscopic Traffic Simulation: An Example Using the Social Force Model [Conference paper, accepted/in press]. In *The Evolving Scholar: BMD 2023 , 5th Edition*. TU Delft OPEN. https://doi.org/10.59490/65a5124da90ad4aecf0ab147
+
+		Supplementary Material
+			Schmidt, C. M., Dabiri, A.; Schulte, F., Happee, R.; Moore, J. K. (2024): Supplementary Material of "Essential Bicycle Dynamics for Microscopic Simulation" - Experiments & Plots [Software].  4TU.ResearchData. https://doi.org/10.4121/574cd504-ad56-4234-8d48-c10931d13204
+
```
Comment (moorepants) [inline content/pages/research/bicycle-dynamics-microsim.rst:96]: Oh, never mind, I see you indented it here, which makes it a `<quote>`. Is that what you want?

File: content/pages/research/bicycle-dynamics-microsim.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+===================================================
+
+:status: hidden
+:date: 2024-05-15
+:collaborators: Jason Moore, Azita Dabiri, Frederik Schulte, Riender Happee 
+:current_researchers: Christoph Schmidt, Anna Marbus
+:research_theme: Cycling Safety
+:template: project
+:summary-sentence: We are investigating the influence of bicycle dynamics on the traffic conflict process to improve traffic simulation models towards predictive safety assessment of infrastructure improvements, smart cycling assistance systems, connectivity, and automated driving. 
+:summary-image: https://objects-us-east-1.dream.io/mechmotum/social-force-example.png
```
Comment (moorepants) [inline content/pages/research/bicycle-dynamics-microsim.rst:12]: This seems to not have a 1:1 aspect ratio. It should be a 400x400 pixel image.

File: content/pages/research/bicycle-dynamics-microsim.rst (hunk 1 above)
Comment (chrismo-schmidt) [reply content/pages/research/bicycle-dynamics-microsim.rst:86]: It's called a "field list" and I used it for not other particular reason then that I liked how the output looked (indented list without bullet points).

**Comment by chrismo-schmidt** [comment]
> One issue, you've written directly to your source branch so I can't clone it without writing over my source branch. You should always create a new branch (branched from the latest source branch) then add your changes.

Ah, I missed that. Should I open a new PR from another branch?

File: content/pages/research/bicycle-dynamics-microsim.rst (hunk 2 above)
Comment (chrismo-schmidt) [reply content/pages/research/bicycle-dynamics-microsim.rst:96]: I experimented quite a bit but didn't really find a satisfying solution. Somehow it should be visually clear that this belongs to the item above.

**Comment by chrismo-schmidt** [comment]
Close in favor of another PR from a different branch.

File: content/pages/research/bicycle-dynamics-microsim.rst (hunk 2 above)
Comment (moorepants) [reply content/pages/research/bicycle-dynamics-microsim.rst:96]: Its fine.

**Pull request closed without merging**